core/lib/Slots.re

```diff
@@ -0,0 +1,78 @@
+module type Elem = {type t('a);};
+
+module type S = {
+  type elem('a);
+  type opaqueElement =
```

Comment:
I got slightly confused about `element`. It's not the reconciler "element", right? This would be more the "content" or the "stored data type" the slots will contain?  🤔

Reply:
great suggestion!

Reply:
Fixed! 
https://github.com/briskml/brisk/pull/27/commits/cc24a0aa719a29b2e18869632218b6668c64a1d3